app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const healthcheck = require('express-healthcheck');
+
+const router = require('./src/routes/index');
+
+const app = express();
+
+app.use('/health', healthcheck());
```

Comment:
Question: why here and not in your router?

Reply:
I added it here because it is middleware so I figured it belonged in app.js but it also makes sense to have it in the router. If it makes more sense to have it in router I will move it there.